### ISSUE TITLE
Support for switching between recent source files

### ISSUE THREAD

**Issue by alffonsse** [smallhadroncollider/taskell#44]
I find myself switching between multiple source/task files for my different projects.
It would be nice to have an in-app way to switch between recent files instead of having to quit the re-open the app with another filename.

I have absolutely zero knowledge regarding Haskell so do you think it would be feasible ?

**Comment by smallhadroncollider** [comment]
It would be perfectly possible to implement, but I feel like it might go against Unix principles. Perhaps a tools like `tmux` would allow this behaviour in a more Unix-y fashion.

Having said that, I can definitely see the use-case. I'll have a think about it. Would appreciate any input from other users.

**Comment by tazimmerman** [comment]
I agree it feels a bit like a violation of the Unix philosophy, but the concept of history is also built into many tools (.bash_history, .viminfo, etc.), so you could have a menu that simply shows the N most recent files taskell has edited (stored in another ini file?) and the user could pick from there.

That said, I use tmux to isolate work in one project from work in another, so I’m comfortable with that approach for taskell as well.

**Comment by janvanveldhuizen** [comment]
I think it is a valid use case. It might be an idea to add a list of taskell files to the config. Then that list can be used to switch between.
